Automattic/jetpack
# PHPCS: Widget Visibility

#### Changes proposed in this Pull Request:

PHPCS cleanup Widget Visibility module files.

#### Jetpack product discussion

Fixes #21979

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

- Make sure the Widget Visibility module is turned on.
- Add a test widget and try changing the visibility settings.
- Look for any errors when interacting with the widget from editor or frontend.
